extensions/pyRevitTools.extension/pyRevit.tab/Modify.panel/edit1.stack/Match.splitpushbutton/Match Properties.pushbutton/script.py

```diff
@@ -76,21 +76,34 @@ def get_source_properties(src_element):
 
     src_type = revit.query.get_type(src_element)
 
+    ignore_bi_parameters = []
+    # sheet number cannot be copied
+    if isinstance(src_element, DB.ViewSheet):
+        ignore_bi_parameters.append(int(DB.BuiltInParameter.SHEET_NUMBER))
```

Comment:
Revit doesn't allow two Sheets to have the same number. Therefore it makes no sense to copy its value